tests/functional/test_eclab_parser.py

```diff
@@ -0,0 +1,406 @@
+"""
+The module tests correct parsing of a mpt files generated by the EC-lab with
+use of the Zilien. The dataset with the files exists as a submodule
+
+NOTE: Test of PEIS technique is skipped, because the Ixdat does not support it.
+
+"""
+from pathlib import Path
+
+from ixdat import Measurement
+
+from pytest import fixture, mark
+
+
+DATA_DIR = Path(__file__).parent.parent.parent / "submodules/ixdat-large-test-files/"
+
+"""
+Data to use in the test.
+
+Format is:
+
+filename_with_technique: {
+  "column_name": ("unit", (first_column_value, last_column_value)),
```

Comment:
Looks great. If one wanted to expand the test a bit one could imagine expanding with number of points or a column content checksum, but I'm not sure if it is necessary.